integration/signing_test.go

```diff
@@ -107,11 +107,11 @@ func (s *SigningSuite) TestSignVerifySmoke(c *check.C) {
 	defer os.Remove(sigOutput.Name())
 	out, err := exec.Command(skopeoBinary, "standalone-sign", "-o", sigOutput.Name(),
 		manifestPath, dockerReference, s.fingerprint).CombinedOutput()
-	c.Assert(err, check.IsNil)
```

Comment:
You could add as third parameter `check.Commentf(out)` so that if it fails you'll get the output also